popup.js

```diff
@@ -0,0 +1,178 @@
+    feauturedImageDeskTop: 'url(\'img/desktop.png\')',
+    technologies: {
+      backend: 'Ruby on rails',
+      style: 'CSS',
+      frontEnd: 'JavaScript',
+      markUP: 'HTML',
+    },
+    linkToLiveVersion: 'https://elad237.github.io/',
+    linkToSource: 'https://github.com/elad237/elad237.github.io',
+  },
+
+  {
+    id: 6,
+    name: 'Multi-Post Stories  Gain+Glory',
+    projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    feauturedImage: 'img/imageplaceholder.png',
+    feauturedImageMobile: 'url(\'img/mobileport.png\')',
+    feauturedImageDeskTop: 'url(\'img/desktop.png\')',
+    technologies: {
+      backend: 'Ruby on rails',
+      style: 'CSS',
+      frontEnd: 'JavaScript',
+      markUP: 'HTML',
+    },
+    linkToLiveVersion: 'https://elad237.github.io/',
+    linkToSource: 'https://github.com/elad237/elad237.github.io',
+  },
+
+];
+
+const generateTechnology = (object) => {
+  let li = ''
+  for(technology in object.technologies){
+    li += `<li class="language"> ${object.technologies[technology]}</li>` 
+  }
+
+  return li;
+}
+const displayProjectCard = (object) => {
+  const projectCard = ` <figure class="project-item">
+  <img class="project-item-img" src=${object.feauturedImage} alt="image placeholder">
+  <figcaption class="project-item-caption">
+      <h3 class="project-item-title">${object.name}</h3>
+      <ul class="project-item-stack">${generateTechnology(object)}</ul>
+      <button class="project-item-btn" id="${object.id}">See project</button>
+  </figcaption>
+</figure>`
+return projectCard;
+};
+
+const displayPopUp = (object) => {
+  const popUp = `<i class="fa fa-times" id="closeddds"></i>
+  <section class="project1 project-text1" id="project1">
+    <div
+      class="featured-image-container"
+      id="images"
+      style="background-image: ${object.feauturedImageDeskTop}"
+    ></div>
+    <div class="project-title1" id="title1">
+      <h4 class="project-name" id="project_name1">${object.name}</h4>
+    </div>
+    <div class="live-source" id="live">
+      <a class="see-project" id="live-demo" href=${object.linkToLiveVersion}>
+        See live<img src="img/liveDemo.svg" />
+      </a>
+      <a class="see-project" id="source" href=${object.linkToSource}>
+        See Source <img src="img/gitIcon.png" />
+      </a>
+    </div>
+    <ul class="langues-use" id="technologypopUp">${generateTechnology(object)}</ul>
+    <p class="project-details" id="details">${object.projectDescription}</p>
+  </section>
+  `
```

Comment:
- Good job on the job done so far 👍 . Your popup still has some little differences from what is in the figma design template. Please reference the image I have attached below 👇 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

# Your project:
-- The close button icon is currently not displaying, please try to fix this issue 😄 
-- Please pay attention to the Project name, it should be aligned with the image (i.e they should have the same margin on the left). Please fix this 👍 .
-- Also, the project buttons on the right should have the same margin on the right side as with the image. Please fix this 🙏 
-- Please ensure that your stick with the general design layout, as regards margins and spacing 👍 👍 
<img width="949" alt="today popup" src="https://user-images.githubusercontent.com/73607512/197334078-8d9f5140-8452-4716-b5d0-4aa67ddd03f1.PNG">

#Figma Design (Desktop version):
<img width="383" alt="popup figma2" src="https://user-images.githubusercontent.com/73607512/197334228-b06f26bb-c5e4-4b66-8500-c7e063232c32.PNG">



(Mobile version): Also, there are some variations in the mobile version of your page, please reference the image of your project attached below:
-- The project buttons (live and source) aren't aligned as they are in the design template, please fix this 👍 👍 
-- Remember to stick to the design layout with regards to the margins and spacing, please fix this 😄 
<img width="150" alt="today popup mobile" src="https://user-images.githubusercontent.com/73607512/197334326-ee537433-5e95-4024-bd94-0519ade047bb.PNG">

## Figma template Mobile:
<img width="161" alt="popup figma mobile" src="https://user-images.githubusercontent.com/73607512/197334347-999ddf64-9eed-4841-abf8-7b9ce309d737.PNG">